package/kedro_viz/server.py

```diff
@@ -415,11 +409,13 @@ def format_pipeline_data(
     # keep track of {data_set_namespace -> layer it belongs to}
     namespace_to_layer = {}
 
-    dataset_to_layer = _construct_layer_mapping(catalog)
+    dataset_to_layer = _construct_layer_mapping()
 
+    # Nodes and edges
     for node in sorted(pipeline.nodes, key=lambda n: n.name):
         task_id = _hash(str(node))
         tags.update(node.tags)
+        _NODES[task_id] = {"type": "task", "obj": node}
```

Comment:
We have both `_NODES` and `nodes` variables in this function. Can we rename one of them as this is really confusing? Maybe `_NODES` -> `_ALL_NODES`?

Reply:
Agree. In Viz, `_NODES` means a graph node/edge in JSON, whereas Kedro nodes mean nodes/pipelines. I can change it to `_JSON_NODES` or something.